examples/deployment/kubernetes/db-deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
           value: "yes"
         - name: MYSQL_USER
           value: test
-        image: us.gcr.io/project-name/db:latest
+        image: gcr.io/${PROJECT_NAME}/db:${TAG}
```

Comment:
Note to self: Our CI infrastructure is in the US, so there's an argument for using the us.gcr.io bucket. However, for the purposes of improving consistency, I'm happy to make it consistently gcr.io for now (especially since that's currently hosted in the US too).